CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ We try only to add languages once they have some usage on GitHub. In most cases
 To add support for a new language:
 
 0. Add an entry for your language to [`languages.yml`][languages]. Omit the `language_id` field for now.
-0. Add a grammar for your language: `script/add-grammar https://github.com/JaneSmith/MyGrammar`. Please only add grammars that have [one of these licenses][licenses].
+0. Add a grammar for your language: `script/add-grammar https://github.com/JaneSmith/MyGrammar` and download its license: `script/licensed`. Please only add grammars that have [one of these licenses][licenses].
```

Comment:
Ah, yes, when I first called `add-grammar` it blew up and I did the rest of the work manually. Thanks, I'll revert this change.